proposals/src/proposal/anchor_update.rs

```diff
@@ -0,0 +1,231 @@
+//! Anchor Update Proposal.
+use crate::{ChainId, ChainType, ProposalHeader};
+
+/// Anchor Update Proposal:
+/// the format of the proposal looks like this:
+/// ```text
+/// ┌────────────────────┬─────────────────┬───────────────┬──────────────────┬────────────────┐
+/// │                    │                 │               │                  │                │
+/// │ ProposalHeader 40B │ SrcChainType 2B │ SrcChainId 4B │ LastLeafIndex 4B │ MerkleRoot 32B │
```

Comment:
```suggestion
/// │ ProposalHeader 40B │ SrcChainType 2B │ SrcChainId 6B │ LastLeafIndex 4B │ MerkleRoot 32B │
```

Recall, chain identifiers should be 6 bytes (2 bytes type + 4 bytes identifier)